packages/client/utils/AtlassianManager.ts

```diff
@@ -264,7 +305,7 @@ export default abstract class AtlassianManager {
       }
       return new Error(json.message)
     }
-    if ('errorMessages' in json) {
+    if ('errorMessages' in json && json.errorMessages.length > 0) {
```

Comment:
This additional checks causes TS errors, I'm still figuring what's the best way of handling that.

Reply:
fixed by adding explicit type guard

Reply:
if errorMessages is empty does that mean it's not an error?

Reply:
Yes, at least now. I don't know what's wrong with Jira but I wasn't even able to make my functionality work so I debugged and this is how the response looked like
```
{
  errorMessages: [],
  errors: {
    customfield_10030: "Field 'customfield_10030' cannot be set. It is not on the appropriate screen, or unknown."
  }
}
```
🤦